controller/lib/core/controller.h

```diff
@@ -47,6 +47,7 @@ class Controller {
   uint64_t breath_id_ = 0;
   BlowerFsm fsm_;
   PID blower_valve_pid_;
+  PID psol_pid_;
```

Comment:
I don't see this PID being actually executed anywhere, looks like you're just using a linear formula instead of the PID. Remove the PID then?

Reply:
To be clear they're for different things.  The linearization in psol.cpp is to translate "I want the psol half open" into a PWM value, say 0.85.  The PID is there to translate "give me 12cmH2O" to "psol should be half open".

The PID isn't being used yet, but it will be.

I can move it to the next PR if you like?

Reply:
If you're about to use it like today, then OK to keep.